MigrationAndImplementationGuide1-4.md

```diff
@@ -1,10 +1,12 @@
-# TRAPI 1.4 Migration and Implementation Guide
+This guide is split into two major sections. The first section focusses on ARA's. The second section focusses on KP's.
```

Comment:
focusses -> focuses
Also apostrophe use? For plural, probably use "ARAs", "KPs", and "CURIEs".